[0.012s] Invoking command in '/home/pragalbhv/ros2-workspaces/sc_ws/build/sc_msg': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/home/pragalbhv/ros2-workspaces/sc_ws/install/sc_pkg:/opt/ros/humble /usr/bin/cmake --build /home/pragalbhv/ros2-workspaces/sc_ws/build/sc_msg -- -j4 -l4
[0.029s] -- Found ament_cmake: 1.3.10 (/opt/ros/humble/share/ament_cmake/cmake)
[0.130s] -- Found geometry_msgs: 4.2.4 (/opt/ros/humble/share/geometry_msgs/cmake)
[0.150s] -- Found rosidl_generator_c: 3.1.5 (/opt/ros/humble/share/rosidl_generator_c/cmake)
[0.152s] -- Found rosidl_adapter: 3.1.5 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.157s] -- Found rosidl_generator_cpp: 3.1.5 (/opt/ros/humble/share/rosidl_generator_cpp/cmake)
[0.164s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.175s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.205s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[0.359s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[0.359s]   execute_process(/usr/bin/python3 -m rosidl_adapter --package-name sc_msg
[0.359s]   --arguments-file
[0.359s]   /home/pragalbhv/ros2-workspaces/sc_ws/build/sc_msg/rosidl_adapter__arguments__sc_msg.json
[0.359s]   --output-dir
[0.359s]   /home/pragalbhv/ros2-workspaces/sc_ws/build/sc_msg/rosidl_adapter/sc_msg
[0.359s]   --output-file
[0.360s]   /home/pragalbhv/ros2-workspaces/sc_ws/build/sc_msg/rosidl_adapter/sc_msg.idls)
[0.360s]   returned error code 1:
[0.360s] 
[0.360s]   Error processing 'int32 SC_COMMAND' of 'sc_msg/Commands': ''SC_COMMAND' is
[0.360s]   an invalid field name.  It should have the pattern
[0.360s]   '^(?!.*__)(?!.*_$)[a-z][a-z0-9_]*$''
[0.360s] 
[0.360s]   Traceback (most recent call last):
[0.360s] 
[0.360s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[0.360s]       return _run_code(code, main_globals, None,
[0.360s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[0.360s]       exec(code, run_globals)
[0.360s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.360s]       sys.exit(main())
[0.360s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[0.360s]       abs_idl_file = convert_to_idl(
[0.360s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 19, in convert_to_idl
[0.360s]       return convert_msg_to_idl(
[0.360s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/msg/__init__.py", line 28, in convert_msg_to_idl
[0.360s]       msg = parse_message_string(package_name, input_file.stem, content)
[0.360s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 519, in parse_message_string
[0.360s]       fields.append(Field(
[0.360s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 354, in __init__
[0.361s]       raise NameError(
[0.361s] 
[0.361s]   NameError: 'SC_COMMAND' is an invalid field name.  It should have the
[0.361s]   pattern '^(?!.*__)(?!.*_$)[a-z][a-z0-9_]*$'
[0.361s] 
[0.361s] Call Stack (most recent call first):
[0.361s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.361s]   CMakeLists.txt:13 (rosidl_generate_interfaces)
[0.361s] 
[0.361s] [0m
[0.361s] -- Configuring incomplete, errors occurred!
[0.361s] See also "/home/pragalbhv/ros2-workspaces/sc_ws/build/sc_msg/CMakeFiles/CMakeOutput.log".
[0.363s] gmake: *** [Makefile:226: cmake_check_build_system] Error 1
[0.365s] Invoked command in '/home/pragalbhv/ros2-workspaces/sc_ws/build/sc_msg' returned '2': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/home/pragalbhv/ros2-workspaces/sc_ws/install/sc_pkg:/opt/ros/humble /usr/bin/cmake --build /home/pragalbhv/ros2-workspaces/sc_ws/build/sc_msg -- -j4 -l4
